website/content/docs/configuration/events/telemetry-data.mdx

```diff
@@ -0,0 +1,586 @@
+        <tr>
+            <td>Timestamp</td>
+        </tr>
+        <tr>
+            <td>Worker ID</td>
+        </tr>
+        <tr>
+            <td rowspan="3">Connections closed</td>
+            <td>Connection ID</td>
+        </tr>
+        <tr>
+            <td>Link (Includes location: org ID, project ID, and internal ID)</td>
+        </tr>
+        <tr>
+            <td>Timestamp</td>
+        </tr>
+        <tr>
+            <td rowspan="5">Connections connected</td>
+            <td>Connection ID</td>
+        </tr>
+        <tr>
+            <td>Connection status</td>
+        </tr>
+        <tr>
+            <td>Connection type</td>
+        </tr>
+        <tr>
+            <td>Link (Includes location: org ID, project ID, and internal ID)</td>
+        </tr>
+        <tr>
+            <td>Timestamp</td>
+        </tr>
+    </tbody>
+</table>
+
+## Groups
+
+<table>
+    <thead>
+        <tr>
+            <th>Events</th>
+            <th>Telemetry fields</th>
+        </tr>
+    </thead>
+    <tbody>
+    <tr>
+            <td rowspan="5">Group add member</td>
+            <td>HCP Boundary user ID</td>
+        </tr>
+        <tr>
+            <td>Link (Includes location: org ID, project ID, and internal ID)</td>
+        </tr>
+        <tr>
+            <td>Parent scope ID</td>
+        </tr>
+        <tr>
+            <td>Scope ID</td>
+        </tr>
+        <tr>
+            <td>Timestamp</td>
+        </tr>
+        <tr>
+            <td rowspan="5">Group remove member</td>
+            <td>HCP Boundary user ID</td>
+        </tr>
+        <tr>
+            <td>Link (Includes location: org ID, project ID, and internal ID)</td>
+        </tr>
+        <tr>
+            <td>Parent scope ID</td>
+        </tr>
+        <tr>
+            <td>Scope ID</td>
+        </tr>
+        <tr>
+            <td>Timestamp</td>
+        </tr>
+        <tr>
+            <td rowspan="5">Group set member</td>
+            <td>HCP Boundary user ID</td>
+        </tr>
+        <tr>
+            <td>Link (Includes location: org ID, project ID, and internal ID)</td>
+        </tr>
+        <tr>
+            <td>Parent scope ID</td>
+        </tr>
+        <tr>
+            <td>Scope ID</td>
+        </tr>
+        <tr>
+            <td>Timestamp</td>
+        </tr>
+        <tr>
+            <td rowspan="5">Groups (Create/delete)</td>
+            <td>HCP Boundary user ID</td>
+        </tr>
+        <tr>
+            <td>Link (Includes location: org ID, project ID, and internal ID)</td>
+        </tr>
+        <tr>
+            <td>Parent scope ID</td>
+        </tr>
+        <tr>
+            <td>Scope ID</td>
+        </tr>
+        <tr>
+            <td>Timestamp</td>
+        </tr>
+    </tbody>
+</table>
+
+## HCP Boundary
+
+<table>
+    <thead>
+        <tr>
+            <th>Events</th>
+            <th>Telemetry fields</th>
+        </tr>
+    </thead>
+    <tbody>
+    <tr>
+            <td rowspan="10">HCP Boundary clusters created</td>
```

Comment:
This is internal as well. only visible to us. customer won't be able to see this.